src/components/SaveGameForm.js

```diff
@@ -3,105 +3,132 @@
     import {useState, useEffect} from 'react'
     import saveLocally from '../lib/saveLocally'
     import loadLocally from '../lib/loadLocally'
-    import Button from './Button'
+    import ButtonPrimary from './Buttons/ButtonPrimary'
     
     SaveGameForm.propTypes = {
         onSubmit: PropTypes.func.isRequired
     }
+
+    const STORAGE_KEY = 'formInputs'
     
     export default function SaveGameForm({onSubmit}) {
        
-        const [formInputs, setFormInput] = useState(loadLocally('formInput') ?? {
-            location: '',
-            date: '',
-            players: '',
-            winner: '',
-            shots: '',
-        })
-    
-        const [inputErrors, setInputErrors] = useState({
+        const [formInputs, setFormInput] = useState(loadLocally(STORAGE_KEY) ?? {
```

Comment:
What about suggesting todays date as default?

Reply:
You have great ideas! :) I'll include this in my backlog.